lldap_config.docker_template.toml

```diff
@@ -45,6 +45,11 @@
 ## For the administration interface, this is the username.
 #ldap_user_dn = "admin"
 
+## Admin email.
+## Email for the admin account. It is only used when initially creating
+## the admin user, and can safely be omitted.
+#ldap_user_email = "admin"
```

Comment:
Can you give a better example value? This is not a valid email.

Reply:
🤦🏻 oops, fixed!